packages/chaire-lib-backend/src/config/server.config.ts

```diff
@@ -47,11 +116,53 @@ const getConfigFilePath = (): string => {
 const configFileNormalized = getConfigFilePath();
 
 // Initialize default server side configuration
-setProjectConfiguration({
+setProjectConfigurationCommon<ServerSideProjectConfiguration>({
     // Initialize runtime directory at the root of the repo
-    projectDirectory: path.normalize(`${__dirname}/../../../../runtime/`)
+    projectDirectory: path.normalize(`${__dirname}/../../../../runtime/`),
```

Comment:
Can we have a comment that says that these define the default values.

Reply:
line 118 is the comment